packages/sku/lib/program/options/stats/stats.option.js

```diff
@@ -0,0 +1,16 @@
+const { Option } = require('commander');
+const statsOption = new Option(
+  '-s, --stats <preset>',
+  'Webpack stats presets',
```

Comment:
I think this could be a bit more descriptive:
```suggestion
  'Webpack stats preset to pass through to webpack',
```